src/modules/Game.class.js

```diff
@@ -6,6 +6,25 @@
  * Feel free to add more props and methods if needed.
  */
 class Game {
+  static get deafultSize() {
```

Comment:
There is a typo in the static method name 'deafultSize'. It should be 'defaultSize'.